org-cyf-itd/content/blocks/step-2/instructions/index.md

```diff
@@ -12,87 +12,78 @@ time= 90
+Which of the prompts below is an appropriate use of AI? 
 
-After leaving school seven years ago, she joined Lawyers 4U as a call handler. She has developed excellent interpersonal skills and has gained experience working under pressure.
+1. Generate a webpage to display information about books in a grid format.
+2. Use CSS Grid to build a webpage to display twenty digital books
+3. Build a homepage for a library website 
+4. I want to learn more about CSS Grid. Please explain the core concepts simply with examples.
 
-##### The Job Description:
+##### Question Two
+A CYF student asks an AI tool to help them understand a complex CSS topic. The information the AI gives doesn't sound right. 
 
-Software for Lawyers Ltd makes case management software. Their software helps lawyers manage caseloads and track time spent. The development team is small and close-knit. They are looking to add a Junior Software Developer to their team.
+Which of the below might be appropriate responses? 
 
-###### Requirements
+1. Googling the topic and look for reliable sources to clarify the point.
+2. Trying to persuade the AI to check again.
+3. Accepting this as the truth because the AI seemed to know what it was talking about.
+4. Asking a volunteer whether the AI is right
+5. Asking another AI tool. 
 
-- A university degree in a relevant subject or software development course
-- Experience with Node.js, Express and React
-- A strong portfolio with solo and group projects
-- Excellent interpersonal skills
-- Experience developing software in an agile team
-- Passionate about programming
+##### Question Three
+A CYF student balances fulltime work and family responsibilities with studying at CYF. They are struggling to keep up with the course. In particular, they are struggling to understand some fundamental concepts. 
 
-##### Nice to Haves
-
-- Experience in the legal sector
-- Some knowledge of Python
-- Experience working in an office-based environment
-
-{{</note>}}
+Which of the below are NOT appropriate responses? 
 
+1. Asking volunteers for guidance at a session
+2. Giving an AI tool their assigned problems and ask it to solve them for them. They then copy and paste the output, making no edits.
+3. Asking an AI tool to explain the concepts they find difficult to understand.
+4. Attempting to solve the problems, then giving up asking an AI tool to fix their code. Then copying and pasting the answer without understanding the changes it made.
+5. Using Google to find articles on the topics they are struggling with. 
+6. Copying another student’s answers.
+7. Sending a direct message to a active volunteer
```

Comment:
It'd be good to be clear that the problem with 2 wasn't just "making no edits" :)

```suggestion
7. Sending a direct message to a active volunteer
8. Giving an AI tool their assigned problems and ask it to solve them for them. They then copy and paste the output, making small edits.
```